students/BrianB/lesson01/sum_double.py

```diff
@@ -0,0 +1,19 @@
+#------------------------------------------
+# Given two int values, return their sum.
+# Unless the two values are the same,
+# then return double their sum.
+#
+# sum_double(1, 2) → 3
+# sum_double(3, 2) → 5
+# sum_double(2, 2) → 8
+#------------------------------------------
+
+def sum_double(a,b):
+    if a != b:
+        return(a+b)
+    elif a==b:
```

Comment:
if a!= b failed, then a == b will always pass (at least for numbers). so don't need the elif.